### PR TITLE
APM-341627

### PR DESCRIPTION
**Pull request by kamiljsokolowski**
This change will:

1. Disable executing the pipeline on PR creation and updates.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -48,7 +49,7 @@ jobs:
         - aws s3 sync $TRAVIS_BUILD_SHARED_STORAGE $AWS_S3_BUILD_SHARED_STORAGE
     - stage: e2e_tests
       name: Execute e2e tests
-      if: branch = master OR type = pull_request OR tag =~ /^release.*$/
+      if: branch = master OR tag =~ /^release.*$/
```
Comment (karolkieglerski) [inline .travis.yml:52]: it should run on all branches not only on master if you remove PR

**Review by karolkieglerski — CHANGES_REQUESTED** [review]
remove branch = master condition

File: .travis.yml (hunk 1 above)
Comment (kamiljsokolowski) [reply .travis.yml:52]: Good point.

Comment (kamiljsokolowski) [reply .travis.yml]: Fixed.

**Review by karolkieglerski — APPROVED** [review]

**Pull request merged**